docs/RELEASE_GIT_GRAPH.md

```diff
@@ -0,0 +1,33 @@
+# Expected git graph
+
+![Release git graph](https://ipfs.io/ipfs/bafkreidcmmgmfjoasjimw66mgbl7vnyvjf4ag4y67urj7ejnkkhm7nxx5m)
+
+## Continued branches
+
+- *master* this is development branch
+- *release* this branch list all the release commits
+  It's important that `git show release~$X` shows version releases first.
+
+## Creating this graph:
```

Comment:
What's the relationship of this document to docs/RELEASE_ISSUE_TEMPLATE.md ?  I don't see any crosslinking.  

Why isn't this info inlined into the release issue template?

I think it would be helpful to state what the purpose of this document is with a "Purpose" section.

Reply:
I haven't compared in detail but at least some of the steps here are also in https://github.com/ipfs/kubo/pull/9024